packages/components/accordion/src/accordion-item.tsx

```diff
@@ -175,13 +219,20 @@ export const AccordionItem = forwardRef<AccordionItemProps, "div">(
     const cloneLabel =
       typeof label === "function"
         ? label({
+            disabled,
+            expanded: open,
             isDisabled,
-            isExpanded: isOpen,
+            isExpanded: open,
```

Comment:
I don't think it's necessary.
delete `isDisabled` and `isExpanded`.

Reply:
`isDisabled` and `isExpanded` are not optional. Therefore, they are required to avoid a type error. I plan to remove them later when working on the `v2.0` branch.